resources/hg-EpisodeOfCare.xml

```diff
@@ -0,0 +1,68 @@
+  <purpose value="A derived profile from [nl-core-EpisodeOfCare](http://nictiz.nl/fhir/StructureDefinition/nl-core-EpisodeOfCare) to provide a version better suited for use in ELZ use cases." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="hg-dataset-020-beta1-20230612" />
+    <uri value="https://decor.nictiz.nl/pub/eerstelijnszorg/hg-html-20230612T223800/ds-2.16.840.1.113883.2.4.3.11.60.103.1.1-2020-10-19T175239.html" />
+    <name value="ART-DECOR dataset ELZ 0.2.0-beta.1 20230612" />
+    <comment value="Relation with the ELZ data set." />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="EpisodeOfCare" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/nl-core-EpisodeOfCare" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="EpisodeOfCare.meta">
+      <path value="EpisodeOfCare.meta" />
+    </element>
+    <element id="EpisodeOfCare.meta.tag">
+      <path value="EpisodeOfCare.meta.tag" />
+    </element>
+    <element id="EpisodeOfCare.meta.tag.userSelected">
+      <path value="EpisodeOfCare.meta.tag.userSelected" />
+      <definition value="Indicates the episode that the general practitioner wants to 'monitor'. These can be either open or closed episodes." />
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-1829" />
+        <comment value="AttentionFlag" />
+      </mapping>
+    </element>
+    <element id="EpisodeOfCare.identifier">
+      <path value="EpisodeOfCare.identifier" />
+      <short value="IdentificationNumber" />
+      <definition value="Globally unique number that identifies the instantiation of the information model. The number is composed of an identification of the issuer organization and a unique number assigned by this organization." />
+      <alias value="Identificatienummer" />
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-1205" />
+        <comment value="Identificatienummer" />
+      </mapping>
+    </element>
+    <element id="EpisodeOfCare.careManager">
+      <path value="EpisodeOfCare.careManager" />
+      <mapping>
+        <identity value="hg-dataset-020-beta1-20230612" />
+        <map value="hg-dataelement-5322" />
+        <comment value="Report Responsible" />
```

Comment:
Hier hoort in de comment HealthProfessional te staan

Reply:
Ik heb 'm ReportResponsible genoemd vanwege de dataset die aangeeft dat dit ZorgEpisodeVerantwoordelijke is.